src/kernel/chainparams.cpp

```diff
@@ -394,6 +396,7 @@ class SigNetParams : public CChainParams {
             .activate = 0x60007600,
             .abandon = 0x40007600,
         };
+        consensus.vDeployments[Consensus::DEPLOYMENT_OPCAT] = SetupDeployment{.activate = 0x62000100, .abandon = 0x42000100, .never = true};
```

Comment:
Can't activate with `.never = true`. Should set `.start` to be the timestamp of the BIN, and `.timeout` to be ten years later. (Should use `.year = 2024, .number = 1, .revision = 0` instead of `.activate` and `.abandon` here and elsewhere as well)

Reply:
Updated. 
And update regtest to 
```
SetupDeployment{
            .activate = 0x62000100,
            .abandon = 0x42000100,
            .always = true
        };
```

please lmk if that is right